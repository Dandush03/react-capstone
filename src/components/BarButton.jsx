import React from 'react';

import toggleFunction from '../javascript/header';

const BarButton = () => (
  <button type="button" className="hamburger" onClick={toggleFunction}>
    <div className="bar1" />
    <div className="bar2" />
    <div className="bar3" />
  </button>
);

export default BarButton;
