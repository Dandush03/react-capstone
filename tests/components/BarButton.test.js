import React from 'react';
import renderer from 'react-test-renderer';
import BarButton from '../../src/components/BarButton';

describe('BarButton', () => {
  test('Render Elements', () => {
    const component = renderer.create(<BarButton />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
