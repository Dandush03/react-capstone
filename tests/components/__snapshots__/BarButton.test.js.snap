// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarButton Render Elements 1`] = `
<button
  className="hamburger"
  onClick={[Function]}
  type="button"
>
  <div
    className="bar1"
  />
  <div
    className="bar2"
  />
  <div
    className="bar3"
  />
</button>
`;
